boutiques_descriptors/micapipe_0_2_2.json

```diff
@@ -0,0 +1,713 @@
+          "list":          false,
+          "value-key": "[OUTPUT_DIR]",
+          "path-template": "[SUBJECT_DIR]_res"
+      }
+  ],
+  "groups": [
+    {
+        "id": "proc_structural_options",
+        "name": "Structural Processing",
+        "members": [
+            "proc_structural",
+            "t1wstr",
+            "uni",
+            "mf"
+        ]
+    },
+    {
+        "id": "proc_surf_options",
+        "name": "Surface Processing",
+        "members": [
+            "proc_surf",
+            "surf_dir",
+            "t1"
+        ]
+    },
+    {
+      "id": "post_structural_options",
+      "name": "Post-structural Processing",
+      "members": [
+          "post_structural",
+          "atlas",
+          "freesurfer"
+      ]
+  },
+  {
+    "id": "gd_module",
+    "name": "Geodesic distance",
+    "members": [
+        "GD"
+    ]
+  },
+  {
+    "id": "flair_module",
+    "name": "FLAIR/T2w processing",
+    "members": [
+        "proc_flair",
+        "flairstr",
+        "regsynth"
+    ]
+  },
+  {
+    "id": "mpc_module",
+    "name": "Microstructural Profile Covariance processing",
+    "members": [
+        "MPC",
+        "microstructural_img",
+        "microstructural_reg",
+        "mpc_acq",
+        "regsynth"
+    ]
+  },
+  {
+    "id": "func_module",
+    "name": "fMRI processing",
+    "members": [
+        "proc_func",
+        "func_main",
+        "func_pe",
+        "func_rpe",
+        "topup",
+        "sesanat",
+        "fix",
+        "nofix",
+        "nsr",
+        "gsr",
+        "droptr",
+        "nofc"
+    ]
+  },
+  {
+    "id": "proc_dwi_options",
+    "name": "DWI Processing",
+    "members": [
+        "proc_dwi",
+        "dwi_main",
+        "dwi_rpe",
+        "dwi_processed",
+        "rpe_all",
+        "regAffine",
+        "dwi_acq",
+        "b0thr",
+        "no_bvalue_scaling",
+        "regsynth"
+    ]
+},
+{
+  "id": "sc_options",
+  "name": "Structural Connectome Processing",
+  "members": [
+      "SC",
+      "tracts",
+      "keep_tck",
+      "autotract",
+      "dwi_acq",
+      "weighted_sc",
+      "tck"
+  ]
+},
+{
+  "id": "qc_modules",
+  "name": "Quality control",
+  "members": [
+      "QC",
+      "QC_subj"
+  ]
+},
+{
+  "id": "cleanup_module",
+  "name": "Cleanup",
+  "members": [
+      "cleanup",
+      "acqstr"
+  ]
+}
+],
+  "tags": {
+      "domain": [
+          "boutiques"
+      ]
+  },
+  "suggested-resources": {
+      "cpu-cores":         10,
+      "ram":               8,
+      "walltime-estimate": 7200
+  },
+  "custom": {
+      "cbrain:readonly-input-files": true,
+      "cbrain:author": "Raul R. Cruces <raul.rodriguezcruces@mcgill.ca>",
+      "cbrain:integrator_modules": {
+        "BoutiquesFileTypeVerifier": {
+            "subject_dir": [ "BidsSubject" ]
+        },
+        "BoutiquesFileNameMatcher": {
+            "subject_dir": "^sub-[a-zA-Z0-9_]+$"
+        },
+        "BoutiquesOutputFileTypeSetter": {
```

Comment:
The Ruby model 'micapipeOutput' doesn't exist; either provide it as a separate file or remove this custom directive. See examples in the 'userfiles' folder. Also, Ruby classes always start with an uppercase letter, so it should be 'MicapipeOutput' (file basename: `micapipe_output.rb`)